code/game/jobs/job/medical.dm

```diff
@@ -121,7 +121,7 @@
 	difficulty = "Ungratifying."
 	selection_color = "#a8b69a"
 	wage = WAGE_PROFESSIONAL
-	alt_titles = list("Soteria Emergency Medical Technician")
+	alt_titles = list("Soteria Paramedic", "Soteria Internal Security Office")
```

Comment:
```suggestion
	alt_titles = list("Soteria Paramedic", "Soteria MedTech")
```
This would be better. They aren't internal security for Soteria, putting 'security' into their job role gives the wrong description.